### DISCUSSION TITLE
Future road map of apiman

### DISCUSSION THREAD

**Discussion by adil695** [apiman/apiman#1318] [General]
Hi,

We are considering apiman for our API gateway and wondering if you have any clear road map with this product in near future as there was no recent major upgrade done. Is this product still going to be supported? Do you offer any SaaS service?

Appreciate any feedback.

Thanks

**Comment by volkflo** [comment]
Hi @adil695,

the project is still under heavy development and we are going to release a new version in the next few days. You can have a look at the [project board](https://github.com/apiman/apiman/projects/1).

Apiman is a non-commercial project and it is supported downstream by multiple commercial sponsors that offer support and SaaS such as Scheer PAS.

You can find more information at the bottom of our [Readme](https://github.com/apiman/apiman#looking-for-support) :)

**Comment by msavy** [comment]
Good post by @volkflo already that we have various folk offering support downstream of Apiman, etc.

Your point on roadmap is a good one, and once we have some spare time we will be refreshing the website and hopefully adding some new roadmap info.

BTW if anyone knows of some good open source software for visualising roadmaps that might be interesting to look at :-).
